generator/ServiceClientGeneratorLib/Generators/Marshallers/RestXmlRequestMarshaller.tt

```diff
@@ -368,4 +372,24 @@ namespace <#=this.Config.Namespace #>.Model.Internal.MarshallTransformations
 <#=new string(' ', level * 4)#>					xmlWriter.WriteEndElement();				
 <#+				
 	}
+#>
+<#+
+	//
+	protected void ProcessNonStructurePayload(Member payload, int level)
+	{
+		if(payload.Shape.IsString)
+		{
+#>
+<#=new string(' ', level * 4)#>request.Content = Encoding.UTF8.GetBytes(<#=this.Operation.RequestPayloadMember.PrimitiveMarshaller#>(publicRequest.<#=this.Operation.RequestPayloadMember.PropertyName#>));
+<#=new string(' ', level * 4)#>request.Headers["Content-Type"] = "text/plain";
```

Comment:
Confirming is okay to always set this to `text/plain`? For example if the payload is a JSON string it is okay to always still treat it as `text/plain`?

Reply:
The smithy docs say if the targeted shape is a string the payload is `text/plain`: https://smithy.io/2.0/aws/protocols/aws-restxml-protocol.html#content-type